WordPress/Classes/ViewRelated/Cells/PickerTableViewCell.swift

```diff
@@ -0,0 +1,133 @@
+import Foundation
+import WordPressShared
+
+
+/**
```

Comment:
The documentation format changed.  The one you're using here doesn't work very well with Xcode + Swift.  This is what comes up when I `alt` + click on this class' name.

<img width="512" alt="screen shot 2015-12-23 at 5 53 00 pm" src="https://cloud.githubusercontent.com/assets/1836005/11984873/0b1124d6-a99e-11e5-8b86-4de819525be0.png">

Here's some [additional info](swift documentation) on the new documentation formats.

Here's an example of how a method documented with the new format looks like:

<img width="613" alt="screen shot 2015-12-23 at 5 57 13 pm" src="https://cloud.githubusercontent.com/assets/1836005/11984932/a267ab2a-a99e-11e5-841a-23aeeb91baba.png">